code/modules/biomatter_manipulation/upgrades.dm

```diff
@@ -0,0 +1,9 @@
+/obj/item/biomatter_transfer_pump
+	name = "Biomatter transfer pump add-on"
+	desc = "A pipe made with gold , silver and glass, created to withstand the corrosion of biomatter. It helps with pumping biomatter in canisters. It can be installed on the biomatter tank platform."
```

Comment:
```suggestion
	desc = "A pipe made with gold, silver and glass, created to withstand the corrosion of biomatter. It can be installed on the biomatter tank platform to speed up with pumping biomatter into canisters."
```